docs/index.md

```diff
@@ -28,8 +37,25 @@ date when the latest version was retrieved, 💡 means the date was inferred by
     <tr>
         <td>
         {% if entry.prefix %}
+            <script type="application/ld+json">
+            {
+                "@context": "https://schema.org",
+                "@type":"Dataset",
+                "http://purl.org/dc/terms/conformsTo": {
+                    "@type": "CreativeWork", 
+                    "@id": "https://bioschemas.org/profiles/Dataset/0.4-DRAFT/"
+                },
+                "name": "{{ entry.name }}",
+                "url": "https://bioregistry.io/{{ entry.prefix }}",
```

Comment:
If this is the URL of the dataset, then I would use `@id` so that your markup here has a declared subject IRI.

Reply:
This isn't the URL of the dataset itself, but rather a third-party page that describes it. Is this valid?

Reply:
We should have an `@id` that identifies the thing that you are describing; this can be your entry URL or some other URL that you have for the dataset. You can then use either `url` or `sameAs` to provide other URLs for the resource.

Reply:
Okay great, then I will change this URL to @id directly

Reply:
Done in f50c862